### DISCUSSION TITLE
MaterialDesign Chips and Content binding

### DISCUSSION THREAD

**Discussion by washburnit** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3291] [Q&A] [answered]
Hi,
I have a Listview where each "GridView.CellTemplate" contains a Chips.
Since I'm loading ListView data from an ItemSource I set the "Content" property of every Chips with a "binding":

```xaml
<GridViewColumn Header="Mese" Width="200">
      <GridViewColumn.HeaderContainerStyle>
          <Style TargetType="{x:Type GridViewColumnHeader}">
              <Setter Property="HorizontalContentAlignment" Value="Left" />
          </Style>
      </GridViewColumn.HeaderContainerStyle>
      <GridViewColumn.CellTemplate>
          <DataTemplate>
              <StackPanel Orientation="Horizontal">
                  <mdix:Chip Content="{Binding meseAnno}" Icon="{Binding iniziale}" Width="180" Foreground="Black" FontFamily="Century Gothic" FontSize="14"/>
              </StackPanel>
          </DataTemplate>
      </GridViewColumn.CellTemplate>
  </GridViewColumn>
```

Data are loaded without errors but every Chips results empty, without text and Icon, simply its shape.
If I replace "Chips" with TextBox data are displayed.
Does Chips support content binding?
Am I doing something wrong?
Thank you, regards.

Roberto

**Comment by washburnit** [comment]
Sorry for code indentation, I was not able to put it in the right way :(

**Reply by nicolaihenriksen** [reply]
@washburnit I fixed the code markdown in your description, you can use three pings and provide a language, then it will format nicely.

Can you show what you are binding to? I mean what are the types returned by the `{Binding meseAnno}` and `{Binding iniziale}`?

**Comment by washburnit** [comment]
Thanks for formatting my code.
The type are both "string".

**Reply by nicolaihenriksen** [reply]
@washburnit It is a little difficult to troubleshoot without a sample to show the issue. Can you provide the sample code? You can also use [Snoop](https://github.com/snoopwpf/snoopwpf) to see what is going on in your visual tree.

Since the properties are both `String`, I think you will probably want to change the icon binding at least. In the code below I am assuming that `{Binding iniziale}` returns a string matching a string representation of a `PackIconKind` value:

```xaml
<mdix:Chip Content="{Binding messoAnno}" ...>
  <mdix:Chip.Icon>
    <mdix:PackIcon Kind="{Binding iniziale}" />
  </mdix:Chip.Icon>
</mdix:Chip>
```

**Comment by washburnit** [comment, marked as answer]
I think I found the problem: it fixed after I updated MaterialDesign to 4.9.0 from 4.8.1.
Maybe a corrupted (previous) installation?
By the way, now it works.
Thank you, regards.
